sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/ConstantFoldingSuite.scala

```diff
@@ -35,6 +35,7 @@ class ConstantFoldingSuite extends PlanTest {
       Batch("AnalysisNodes", Once,
         EliminateSubQueries) ::
       Batch("ConstantFolding", Once,
+        OptimizeIn,
         ConstantFolding,
         BooleanSimplification) :: Nil
```

Comment:
The initial plan here was to only include the rules we are testing.  I'm not sure if its true here, but sometimes including all the rules make it harder to test since then you need something that won't be optimized away by unrelated rules.